packages/frontend/src/contexts/AppBarContext.tsx

```diff
@@ -23,7 +25,8 @@ export const AppBarContext = React.createContext({
 export type TUserRight = { canRead: boolean; canWrite: boolean };
 
 export const AppBarContextProvider = (props: any) => {
-  const [openDrawer, setOpenDrawer] = useState(false);
+  const [openMiniDrawer, setOpenMiniDrawer] = useState(false);
+  const [openDesktopDrawer, setOpenDesktopDrawer] = useState(localStorage.getItem('desktopDrawerOpen') === 'true');
```

Comment:
> What are the benefits of `openDesktopDrawer` that is saved to local storage, for example compared to `const [openDesktopDrawer, setOpenDesktopDrawer] = useState(true)`?

It's to persist the state of the drawer (open or close) between switching pages and page refreshes, since the local state gets cleared on every refresh